src/woo_publications/templates/admin/change_owner.html

```diff
@@ -0,0 +1,91 @@
+    <link rel="stylesheet" type="text/css" href="{% static 'admin/css/forms.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'admin/css/admin-index.css' %}">
+{% endblock %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} change-owner-confirmation change-owner-selected-confirmation{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+&rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; {% translate 'Change owner of multiple objects' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <fieldset class="module aligned ">
+        <p>{% blocktranslate %}Are you sure you want to change the owner of the selected {{ objects_name }}? All of the following objects will have their owner changed:{% endblocktranslate %}</p>
+        <ul>
+            {% for changeable_object in changeable_objects %}
+                <li>{{ changeable_object }}</li>
+            {% endfor %}
+        </ul>
+        <hr />
+    </fieldset>
+
+    <fieldset class="module aligned ">
+        <p>
+        {% blocktranslate %}
+        Select a new owner in the field named "Owner".<br>
+        Cannot find the user you are looking for?<br>
+        Use the fields named "Identifier" and "Name" instead!
+        {% endblocktranslate %}
+        </p>
+
+        <form method="post">{% csrf_token %}
+            <div>
+                {% if form.errors %}
+                    <p class="errornote">
+                    {% blocktranslate %}Please correct the error(s) below.{% endblocktranslate %}
+                    </p>
+                    {{ form.non_field_errors }}
+                {% endif %}
+
+                <fieldset class="module aligned">
+                    {% for field in form %}
+                        <div class="form-row field-{{ field.name }}">
+                            {% if field.errors %}
+                                {{ field.errors }}
+                            {% endif %}
+                            <div>
+                                <div class="flex-container">
+                                    <label class="required" for="id_{{ field.name }}">{{ field.label }}: </label>
+                                    {{ field }}
+                                </div>
+                                {% if field.help_text %}
+                                    <div class="help" id="id_{{ field.name }}_helptext">
+                                        <div>{{ field.help_text }}</div>
+                                    </div>
+                                {% endif %}
+                            </div>
+                        </div>
+                    {% endfor %}
```

Comment:
Might be better to not do this dynamically so we can have some text in between the `Owner` field and `Identifier`-`Name` field? To explain that the 2 below are to add a new user.

Reply:
Yeah, I'd separate those groups. Perhaps a fieldset for the dropdown and a fieldset for the text fields is better.